### PR TITLE
chore: add quickstart example with java

### PR DESCRIPTION
**Pull request by danielbdias**
This PR adds a quick start example using a Java API.

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: examples/quick-start-java/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Quick Start - Ruby on Rails app with OpenTelemetry and Tracetest
+
+> [Read the detailed recipe for setting up OpenTelemetry Collector with Tractest in our documentation.](https://docs.tracetest.io/examples-tutorials/recipes/running-tracetest-without-a-trace-data-store)
+This is a simple quick start on how to configure a Ruby app (with [Rails](https://rubyonrails.org/) framework) to use OpenTelemetry instrumentation with traces, and Tracetest for enhancing your e2e and integration tests with trace-based testing.
```
Comment (mathnogueira) [inline examples/quick-start-java/README.md:4]: I know you like Rails, but I think this should be about Java :rofl:

**Review by mathnogueira — APPROVED** [review]
Other than the readme, it looks good

**Review by xoscar — APPROVED** [review]

**Comment by xoscar** [comment]
I tested it and looks good, great job @danielbdias!

File: examples/quick-start-java/README.md (hunk 1 above)
Comment (danielbdias) [reply examples/quick-start-java/README.md]: 🙃 
(Thanks for this catch!)

**Pull request merged**